README.md

```diff
@@ -22,18 +22,24 @@ Inherit `rubocop.yml` and add specific settings for each repository like `AllCop
 inherit_from:
   - https://raw.githubusercontent.com/pixta-dev/pixta-rubocop/master/rubocop.yml
 
+# Normally additional configs of AllCops/Include and AllCops/Ecxlude override to RuboCops's defaults.
+# But you'd like to apply both RuboCops's defaults and addtional configs of AllCops/Include and AllCops/Ecxlude.
+# ref: https://docs.rubocop.org/en/stable/configuration/#unusual-files-that-would-not-be-included-by-default
+inherit_mode:
+  merge:
+    - Include
+    - Exclude
+
 AllCops:
   TargetRubyVersion: TODO
   Include:
-    - '**/Rakefile'
     - '**/config.ru'
```

Comment:
[nits]
this file also seems listed in the default config. so it can be deleted, i think.
ref. https://github.com/rubocop-hq/rubocop/blob/c50e7473f8627dd276cb7656bcd1953cb74bb243/config/default.yml#L31